contentService/src/main/java/com/kovan/ContentServiceMain.java

```diff
@@ -2,9 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
-@SpringBootApplication
+@SpringBootApplication(exclude = DataSourceAutoConfiguration.class)
```

Comment:
why you added exclude DataSourceAutoConfiguration class ?

Reply:
exclude DataSourceAutoConfiguration removed.